WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "4.13.0-beta.4"
+  s.version       = "4.13.0-beta.5"
```

Comment:
I think this should be `4.14.0-beta.1`. We just finished a release cycle, so I expect `4.13` has been released. (Maybe this is still beta because @jkmassel just hasn't update all the pods yet.)

Reply:
It's updated now.